sdk-lib/src/main/java/cash/z/ecc/android/sdk/jni/RustBackend.kt

```diff
@@ -56,18 +56,15 @@ class RustBackend private constructor() : RustBackendWelding {
         )
     }
 
-    override suspend fun initAccountsTable(vararg keys: UnifiedViewingKey): Boolean {
-        val extfvks = Array(keys.size) { "" }
-        val extpubs = Array(keys.size) { "" }
+    override suspend fun initAccountsTable(vararg keys: UnifiedFullViewingKey): Boolean {
+        val ufvks = Array(keys.size) { "" }
```

Comment:
A stylistic choice, but we can probably inline this now and drop the forEach below.  The lambda for the Array() constructor provides an index as `it` and the return value initializes that index.

`Array(keys.size) { keys[it].encoding }`

Reply:
I stuck close to the existing style because I'm not a Kotlin dev 😄 We should just clean this up across the board in a subsequent PR.

Reply:
I've resolved this with an additional commit.